docs/location_changes.md

```diff
@@ -1,49 +1,56 @@
 # When a location changes in Alma:
 
-### 1. Update the local dev environment:
+## 1. Update the local dev environment:
 
-Load the rails console
-1.`bundle exec rails c`
+1. Update the necessary files in https://github.com/pulibrary/bibdata/tree/main/config/locations. 
+   * If there is a new holding location then update the holding_locations.json file https://github.com/pulibrary/bibdata/blob/main/config/locations/holding_locations.json with the appropriate information. 
+   * If there is a new library then update the libraries.json file https://github.com/pulibrary/bibdata/blob/main/config/locations/libraries.json with the new library following the pattern in the file. 
+   * If there is a new delivery location then update the delivery_locations.json file https://github.com/pulibrary/bibdata/blob/main/config/locations/delivery_locations.json with the appropriate information.
 
-Delete the existing data from the location tables and repopulate them by pulling data from Alma.
-2.`LocationDataService.delete_existing_and_repopulate`
+2. Load the rails console:  
+   `bundle exec rails c`
 
-Generate the following files: marc_to_solr/translation_maps/location_display.rb and marc_to_solr/translation_maps/locations.rb
-3.`LocationMapsGeneratorService.generate`
+3. Delete the existing data from the location tables and repopulate them using the config files.
+  `LocationDataService.delete_existing_and_repopulate`
 
-Copy the content from the generated .rb files into the .tmpl.rb files
-4.`cp marc_to_solr/translation_maps/location_display.rb marc_to_solr/translation_maps/location_display.rb.tmpl`
-`cp marc_to_solr/translation_maps/locations.rb marc_to_solr/translation_maps/locations.rb.tmpl`
+4. Generate the following files: marc_to_solr/translation_maps/location_display.rb and marc_to_solr/translation_maps/locations.rb
+  `LocationMapsGeneratorService.generate`
 
-Load locally the rails server; Go to `localhost:<portnumber>/locations/holding_locations` and make sure that the locations have been updated.
+5. Copy the content from the generated .rb files into the .tmpl.rb files  
```

Comment:
```suggestion
5. Exit the rails console, then copy the content from the generated .rb files into the .tmpl.rb files  
```

Reply:
Console is not needed anymore. I added the rake task. I didn't update the wording with exit the console.